k9mail/src/main/java/com/fsck/k9/fragment/MessageListAdapter.java

```diff
@@ -168,6 +174,24 @@ public void bindView(View view, Context context, Cursor cursor) {
         boolean answered = (cursor.getInt(ANSWERED_COLUMN) == 1);
         boolean forwarded = (cursor.getInt(FORWARDED_COLUMN) == 1);
 
+        Keyword firstTag = null;
```

Comment:
Put this code in a separate method.

Reply:
Here it is: `MessageListAdapter.getFirstTag()`.